includes/admin/assets.php

```diff
@@ -143,6 +143,7 @@ public function load() {
 			$conditions = array(
 				in_array( $post_type, (array) $post_types ),
 				$screen->id == 'widgets',
+				is_admin() && isset( $_GET['action'] ) && 'edit' === $_GET['action'],
```

Comment:
@damonsharp how confident are you that this doesn't prevent the JS from loading on a page where it's needed?

Reply:
@zachwills actually, this was too restricted and the error was still happening on new post screen (pages, etc.), so I refactored. good catch. my testing seems okay now.